.github/workflows-source/ci-main.yml

```diff
@@ -22,12 +22,11 @@ permissions: read-all
 env:
   CI_COMMIT_SHA: ${{ github.sha }}
   CI_JOB_NAME: ${{ github.job }}
-  CI_JOB_ID: ${{ github.job }} # github does not expose this variable https://github.com/orgs/community/discussions/8945
+  CI_JOB_ID: ${{ github.run_id }}
```

Comment:
Better to have `CI_RUN_ID: ${{ github.run_id }}` as the name you have now is technically wrong and can be confusing.